tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow EDTF Date String."""
+
+import unittest
+from datetime import date
+
+import pytest
+from edtf.parser.grammar import ParseException, level0Expression
+from marshmallow import ValidationError, fields
+
+from marshmallow_utils.fields import edtfdatestring
+
+
+def test_deserialize():
+
+    f = edtfdatestring.EDTFDateString()
```

Comment:
The usage of EDTFDateString will always be via a definition in a schema, thus it's better to test it directly in the way it's intended to be used. I.e. somethink like:

```python
class MySchema(Schema):
    f = EDTFDateString()


assert MyShema().dump({'f': '2020-08'))  == {'f': '2020-08'}
```

Reply:
Just updated the code to use schemas!